packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -543,7 +553,7 @@ export interface AutoScalingGroupProps extends CommonAutoScalingGroupProps {
   /**
    * Type of instance to launch
    *
-   * `launchTemplate` must not be specified when this property is specified.
+   * Cannot specify this property when `launchTemplate` or `mixedInstancesPolicy` are specified
```

Comment:
This comment stands for each instance here (I won't repeat it because that's just kind of annoying), but I think we actually want the original style comment of:
```
`whatever` must not be specified when `whatever-else` is specified.
```